packages/ui5-proxy-middleware/test/base/proxy.test.ts

```diff
@@ -0,0 +1,100 @@
+import { ui5Proxy } from '../../src';
+import * as hpm from 'http-proxy-middleware';
+import * as utils from '../../src/base/utils';
+import { ToolsLogger } from '@sap-ux/logger';
+
+describe('ui5Proxy', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    test('ui5Proxy: default params', () => {
+        const createProxyMiddlewareSpy = jest.spyOn(hpm, 'createProxyMiddleware').mockImplementation(jest.fn());
+        const proxyRequestHandlerSpy = jest.spyOn(utils, 'proxyRequestHandler').mockImplementation(jest.fn());
+        const config = {
+            path: '/mypath',
+            url: 'https://example.example',
+            version: '1.0.0'
+        };
+
+        const req = {
+            headers: {
+                accept: 'text/html',
+                'accept-encoding': 'gzip'
+            }
+        };
+
+        ui5Proxy(config);
+        expect(createProxyMiddlewareSpy).toHaveBeenCalledTimes(1);
+        const defaultFilterFn = createProxyMiddlewareSpy.mock.calls[0][0];
+        expect(defaultFilterFn).toEqual(expect.any(Function));
+        const expectedOptions = createProxyMiddlewareSpy.mock.calls[0][1];
+        expect(expectedOptions?.changeOrigin).toBeTruthy();
+        expect(expectedOptions?.selfHandleResponse).toBeTruthy();
+        expect(expectedOptions?.target).toEqual('https://example.example');
+        expect(expectedOptions?.onProxyReq).toEqual(expect.any(Function));
+        expect(expectedOptions?.onProxyRes).toEqual(expect.any(Function));
+        expect(expectedOptions?.pathRewrite).toEqual({ '/mypath': '/1.0.0/mypath' });
```

Comment:
Can these assertions be simplified using something like [`expect.objectContaining()`](https://jestjs.io/docs/expect#expectobjectcontainingobject) and [`expect.toHaveBeenCalledWith()`](https://jestjs.io/docs/expect#tohavebeencalledwitharg1-arg2-)